projects/_posts/2016-08-07-Mapping.md

```diff
@@ -0,0 +1,253 @@
+---
+enable_x3d_support: true
+author: [ "@bmegli" ] 
+programming_language: [ "C/C++/C#/HLSL" ] 
+thumbnail_override: "/images/projects/2016-08-07-Mapping/project_thumbnail.png"
+
+youtube_video_id: zqzevoKpPjQ # The video ID of the YouTube video to be displayed with this post
+
+project_homepage_url: "https://github.com/bmegli/ev3dev-mapping" 
+source_code_url: "https://github.com/bmegli/ev3dev-mapping" 
+#building_instructions_url: "" # how to build the model out of LEGO (*not* how to build the source code)
```

Comment:
No worries. As an aside, did you print the deck plate that you use in the phaseII model or is it available somewhere?